helpers/stringset/string_set.go

```diff
@@ -7,8 +7,24 @@
 
 package stringset
 
+// func NewStringSet(initial ...string) StringSet {
```

Comment:
?

Reply:
Sorry, thought I might need an initializer, but then forgot about it.